xml/admin_upgrade.xml

```diff
@@ -256,4 +256,213 @@ value: value)}'</command></screen>
+      number of nodes by replacing the value for the (<literal>--batch</literal>)
+      parameter.
+     </para>
+    </note>
+    <screen>
+&prompt.user;<command>docker exec -i $(docker ps | grep salt-master | awk '{print $1}') \
+salt --batch <replaceable>10</replaceable> -P "roles:(admin|kube-(master|minion))" \
+cmd.run "transactional-update salt migration -n" \
+| tee transactional-update-migration.log</command>
+
+Executing run on ['5f6688bbeac94d2ab5c4330dc7043fb2',
+                  'c3afd049edbe43afb4e2e5913a88291b',
+                  '5bf346291a18406290886c2e2f7c3e3f',
+                  'admin']
+
+5bf346291a18406290886c2e2f7c3e3f:
+    
+    
+    Executing 'zypper --root /tmp/tmp.vbaqUwrLIh --non-interactive  refresh'
+    
+    Retrieving repository 'SUSE-CAASP-ALL-Pool' metadata [...done]
+    Building repository 'SUSE-CAASP-ALL-Pool' cache [....done]
+    Retrieving repository 'SUSE-CAASP-ALL-Updates' metadata [....done]
+    Building repository 'SUSE-CAASP-ALL-Updates' cache [....done]
+    All repositories have been refreshed.
+    Upgrading product SUSE CaaS Platform 3.0 x86_64.
+
+[ SNIP ... ]
+
+    done
+jid:
+    20180807122253512832
+retcode:
+    0
+    </screen>
+    <para>
+     During the procedure the nodes will be switched to the new release channel
+     for v3, available updates are downloaded and installed, services and 
+     applications are reconfigured and brought up in a orderly fashion.
+    </para>
+    <para>
+     This operation will produce a lot of output for each node. The entire output
+     is mirrored to a log file <filename>transactional-update-migration.log</filename>
+     to the current working directory. This log file can be very helpful should
+     any of the update operations fail.
+    </para>
+   </sect2>
+   
+   <sect2 xml:id="sec.deploy.upgrade.caasp2.reboot">
+    <title>Reboot cluster nodes from &dashboard;</title>
+    <para>
+     To complete the procedure, you must reboot the cluster nodes. To do this 
+     properly, use &dashboard; to restart the nodes.
+    </para>
+    <procedure>
+     <step>
+      <para>
+       Log in to &dashboard;.
+      </para>
+     </step>
+     <step>
+      <para>
+       Reboot the Admin node as described in 
+       <xref linkend="transactional.updates.installation" />.
+      </para>
+     </step>
+     <step>
+      <para>
+       Reboot the remaining nodes as described in 
+       <xref linkend="transactional.updates.installation" />.
+      </para>
+     </step>
+    </procedure>
+   </sect2>
+  
+  <sect2 xml:id="sec.deploy.upgrade.caasp2.troubleshooting">
+   <title>Troubleshooting</title>
+   <para>
+    In case the upgrade fails, please perform the support data collection by 
+    running <command>supportconfig</command> on the affected nodes. Provide the 
+    resulting files including the <filename>transactional-update-migration.log</filename>
+    to SUSE Support.
```

Comment:
would it make sense to advise for a rollback here aside of the supportconfig suggestion?

Reply:
@MaximilianMeister Can you link me to rollback instructions? I thought the same thing when writing this but I don't have any information on how to perform a rollback.

Reply:
@kiall I'm ok with this reasoning at this stage but customers will require working rollback for any operation. If we don't know how to do it without breaking stuff, we need to get that figured out. (@ajaeger)

Reply:
In this case (v2->v3 update), rollback would need to be done for the whole cluster. (CC @thkukuk) and that looks tricky.

In other places we should document rollback.